File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.java
import java.util.*;
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        
        for(int i=0; i<num_list.length; i++){
            int n = num_list[i];
            
            while(true){
                if(n/2==0) break;
                
                if(n%2 == 0){
                    n/=2;
                    answer++;
                }else{
                    n= (n-1)/2;
                    answer++;
                }
                
            }
            
            
        }
        
        return answer;
    }
}

File: 프로그래머스/0/181880. 1로 만들기/1로 만들기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

// num_list_len은 배열 num_list의 길이입니다.
int solution(int num_list[], size_t num_list_len) {
    int answer = 0;
    
    for (int i=0; i<num_list_len; i++)
    {
        int n = num_list[i];
        
        while(n/2!=0)
        {
            if(n%2==0)
            {
                n = n/2;
                answer++;
            }
            else{
                n = (n-1) / 2;
                answer++;
            }
        }
    }
    
    return answer;
}